doc/cephadm/host-management.rst

```diff
@@ -415,6 +415,72 @@ run on the host.
   profiles with the same name are not overwritten.
 
 
+Custom Logrotate Configs
+========================
+
+Cephadm can write out custom logrotate configs for either the
```

Comment:
But not OSD, mgr, MDs, mon, or other daemon logs?

Reply:
the "cluster logs" would include the logs for every type of core ceph daemon. We don't have separate logrotate configs per daemon type.

Reply:
Got it, thanks - I was thinking of the cluster log as in period pg stat.  Would it be reasonable to change this to say "daemon logs"?

Reply:
sure, I was going with that name because the actual logrotate template is `cluster.logrotate.config.j2`, but we're referring to the logs in a more general sense in this sentence so that should be fine

Reply:
Ack, thanks.  My concern was only that admins might not understand what is and isn't covered.  Personally I'd do everything on the same schedule anyway, but that's me.